package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "echo 'Warning: TODO - Define Linting.'",
     "pretest-ci": "npm-run-all -s lint bootstrap",
     "seed": "cross-env DEBUG=fcc:* node ./tools/scripts/seed/seedChallenges",
+    "seed-auth-user": "cross-env DEBUG=fcc:* node ./tools/scripts/seed/seedAuthUser",
```

Comment:
I could use some guidance on how to set this part up.

Reply:
@raisedadead

Reply:
Sure, I'll propose a few changes in the areas, which you can add or accept.